bciers/apps/reporting/src/app/utils/getReportNeedsVerification.ts

```diff
@@ -1,6 +1,15 @@
 import { actionHandler } from "@bciers/actions";
+import { getReportingOperation } from "@reporting/src/app/utils/getReportingOperation";
+import { OperationTypes } from "@bciers/utils/src/enums";
 
 export async function getReportNeedsVerification(reportVersionId: number) {
+  // 🚀 Fetch the operation associated with the specified version ID
+  const reportOperation = await getReportingOperation(reportVersionId);
+  // Check for EIO bypass
+  const operationType = reportOperation?.operation_type;
+  if (operationType === OperationTypes.EIO) {
+    return true;
+  }
```

Comment:
This should happen on the server, the `ReportVerificationService.get_report_needs_verification(...)` method called through the API below is there for exactly that case